universal-login-relayer/lib/startRelayer.js

```diff
@@ -1,7 +1,11 @@
 import Relayer from './relayer';
+import knex from 'knex';
+import knexConfig from '../knexfile';
 require('dotenv').config();
 
 const config = require('./config/relayer');
 
-const relayer = new Relayer(config);
+const database = knex(knexConfig[`${process.env.NODE_ENV}`]);
```

Comment:
👍

Reply:
also I would propose to extract some helper/utils functions:
```js
function getNodeEnv() {
  return process.env.NODE_ENV || 'development';
}

function getKnexConfig() {
  return knexConfig[getNodeEnv()];
}

function getKnex() {
  retrun knex(getKnexConfig());
}
```

Feels somehow good.

Reply:
✅